packages/synapse-react-client/src/components/EntityFinder/EntityFinderModal.tsx

```diff
@@ -56,8 +56,16 @@ export const EntityFinderModal = (props: EntityFinderModalProps) => {
           }
         }}
         content={
-          <Box width="min-content">
-            <Typography variant="body1" mb="10px">
+          <Box>
+            <Typography
+              variant="body1"
+              mb="10px"
+              sx={{
+                // Do not allow the help text to expand the dialog width
+                minWidth: '100%',
+                width: 0,
+              }}
+            >
```

Comment:
The `width: min-content` style was designed to prevent this scenario, where the paragraph text would expand the dialog width even when it should be narrow:

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/44e15bec-1969-4efd-a456-b008dcca337d)

The styles above directly applied to the paragraph solve this in a more predictable way that doesn't affect other EntityFinder elements